core/templates/dev/head/domain/story/StoryContentsObjectFactory.js

```diff
@@ -17,14 +17,21 @@
  * story contents domain objects.
  */
 
-oppia.factory('StoryContentsObjectFactory', ['StoryNodeObjectFactory',
-  function(StoryNodeObjectFactory) {
+oppia.factory('StoryContentsObjectFactory', [
+  'StoryNodeObjectFactory', 'NODE_ID_PREFIX',
+  function(StoryNodeObjectFactory, NODE_ID_PREFIX) {
     var StoryContents = function(initialNodeId, nodes, nextNodeId) {
       this._initialNodeId = initialNodeId;
       this._nodes = nodes;
       this._nextNodeId = nextNodeId;
     };
 
+    var _incrementNodeId = function(nodeId) {
```

Comment:
Actually -- let's call this getIncrementedNodeId or similar. "_incrementX" doesn't have semantics of "return".